packages/react-server-cli/src/buildWebpackCompilers.js

```diff
@@ -324,37 +204,21 @@ module.exports = {
 	const routesContent = routesOutput.join("");
 	// make a unique file name so that when it is required, there are no collisions
 	// in the module loader between different invocations.
-	const routesMD5 = crypto.createHash('md5').update(routesContent).digest("hex");
-	const routesFilePath = `${workingDirAbsolute}/routes_${isClient ? "client" : "server_" + routesMD5}.js`;
+	//const routesMD5 = crypto.createHash('md5').update(routesContent).digest("hex");
+	const routesFilePath = `${workingDirAbsolute}/routes_${isClient ? "client" : "server"}.js`;
 	fs.writeFileSync(routesFilePath, routesContent);
 
 	return routesFilePath;
 }
 
-
-// the page value for routes.routes["SomeRoute"] can either be a string for the default
-// module name or an object mapping format names to module names. This method normalizes
-// the value to an object.
-function normalizeRoutesPage(page) {
-	if (typeof page === "string") {
-		return {default: page};
-	}
-	return page;
-}
-
 // writes out a bootstrap file for the client which in turn includes the client
 // routes file. note that outputDir must be the same directory as the client routes
 // file, which must be named "routes_client".
-function writeClientBootstrapFile(outputDir, opts) {
-	var outputFile = outputDir + "/entry.js";
+function writeClientBootstrapFile(outputFile, opts) {
 	fs.writeFileSync(outputFile, `
 		if (typeof window !== "undefined") {
 			window.__setReactServerBase = function(path) {
-				// according to http://webpack.github.io/docs/configuration.html#output-publicpath
-				// we should never set __webpack_public_path__ when hot module replacement is on.
-				if (!module.hot) {
-					__webpack_public_path__ = path;
-				}
```

Comment:
Yes, they don't.  Here are the references:

http://code.fitness/post/2016/02/webpack-public-path-and-hot-reload.html#overriding-the-public-path:12018ae465828e3168201053550649e1

https://github.com/webpack/webpack/issues/1650